src/lib/pull-request.ts

```diff
@@ -1,10 +1,11 @@
 import fs from "fs";
 import path from "path";
-import moment from 'moment';
 import auth, { ghauthData } from "./auth";
 import { GraphQLClient } from "graphql-request";
 import chalk from "chalk";
 
+var moment = require('moment-business-days');
```

Comment:
`const`

Reply:
Although I think you could also use `import * as moment from 'moment-business-days';`